src/include/miopen/solver/implicitgemm_ck_util.hpp

```diff
@@ -122,14 +151,17 @@ ConvSolution InitInvokerFactoryNHWC(const ExecutionContext&,
                 const auto& data_ctx = primitive_parameters.CastTo<CastType>();
                 auto argument_ptr    = ck_args.MakeArgPtr(sh_conv_ptr, data_ctx.tensors);
                 auto invoker_ptr     = sh_conv_ptr->MakeInvokerPointer();
-
-                const auto enable_profiling = handle.IsProfilingEnabled();
-                float elapsed_time =
-                    invoker_ptr->Run(argument_ptr.get(), {handle.GetStream(), enable_profiling});
-                if(enable_profiling)
                 {
-                    handle.ResetKernelTime();
-                    handle.AccumKernelTime(elapsed_time);
+#ifdef WORKAROUND_CK_ISSUE_1184
+                    HipEventProfiler pfr(handle);
+#endif
```

Comment:
We do not have "normal" implementation here yet, and therefore disabling the workaround should lead to build errors. The simplest way to achieve this:
```suggestion
                    HipEventProfiler pfr(handle);
```